src/i_3dsound.c

```diff
@@ -86,11 +86,14 @@ static void CalcListenerParams(const mobj_t *listener,
     lis->position[1] = FIXED_TO_ALFLOAT(player->viewz);
     lis->position[2] = FIXED_TO_ALFLOAT(-listener->y);
 
-    if (oal_use_doppler)
+    if (oal_use_doppler && listener->interp == true)
```

Comment:
Why do you bind this to interpolation now?

Reply:
In special cases, like when the player teleports, I think it makes sense to ignore the velocity for a tic. Otherwise there can be extreme values.

```
reactiontime 0 interp 1 dx 0.004 dy -9.795 dz 0.000 <-- running toward teleporter
reactiontime 0 interp 1 dx 0.004 dy -10.439 dz 0.000
reactiontime 0 interp 1 dx 0.004 dy -11.023 dz 0.000
reactiontime 0 interp 1 dx 0.004 dy -11.552 dz 0.000
reactiontime 18 interp 0 dx -0.033 dy -359.669 dz 0.000 <-- teleported
reactiontime 17 interp 1 dx 0.000 dy 0.000 dz 0.000
reactiontime 16 interp 1 dx 0.000 dy 0.000 dz 0.000
reactiontime 15 interp 1 dx 0.000 dy 0.000 dz 0.000
reactiontime 14 interp 1 dx 0.000 dy 0.000 dz 0.000
```